worlds/timespinner/Options.py

```diff
@@ -292,18 +277,20 @@ class ShowDrops(Toggle):
     display_name = "Show Bestiary Item Drops"
 
 
+class EnterSandman(Toggle):
+    "Ancient Pyramid is unlocked by the Twin Pyramid Keys, but the final boss door opens if you have all 5 Timespinner pieces"
```

Comment:
Suggestion:
"The Ancient Pyramid is unlocked by the Twin Pyramid Key, but the final boss door only opens if you have all 5 Timespinner pieces"

Reply:
Fine